jax/experimental/ode.py

```diff
@@ -161,6 +161,11 @@ def odeint(func, y0, t, *args, rtol=1.4e-8, atol=1.4e-8, init_step=-1., mxstep=n
 def _odeint_wrapper(func, rtol, atol, init_step, mxstep, y0, ts, *args):
   y0, unravel = ravel_pytree(y0)
   func = ravel_first_arg(func, unravel)
+  if init_step:
+    init_step = init_step
+  else:
```

Comment:
Slightly cleaner / less redundant:
```suggestion
  if init_step is None:
```

(It's also better to check against `None` explicitly for default values, otherwise this would entail boolean evaluation of `init_step` which would fail for JAX tracers)